spec/lib/api/v3/work_packages/work_package_representer_spec.rb

```diff
@@ -35,12 +35,15 @@
     FactoryGirl.create(:user,  member_in_project: project, member_through_role: role)
   end
   let(:current_user) { member }
+  let(:current_user) { member }
```

Comment:
I prepared this as a [duck](http://blog.codinghorror.com/new-programming-jargon/) for you... Congratulations. You found it!